### DISCUSSION TITLE
referenceSystem

### DISCUSSION THREAD

**Discussion by AdrianKriger** [cityjson/specs#97] [Q&A] [answered]
Hi;

When creating a CityJSON, defining a:
```"referenceSystem": "https://www.opengis.net/def/crs/EPSG/0/32734", ```
property and attempt to load it later:
```python
cm = cityjson.load("./result/citjsnCl_57.json")
print(cm)
```
result -> ```ValueError: invalid literal for int() with base 10: 'ttps://www.opengis.net/def/crs/EPSG/0/32734'```

**Comment by balazsdukai** [comment, marked as answer]
I transferred your question to the cjio repository, because it looks like an issue related to cjio and not related to the cityjson specifications. See https://github.com/cityjson/cjio/issues/128 .
I'm closing this discussion.

**Reply by AdrianKriger** [reply]
You are very kind. Thank you. 

Where would I raise a question about `"+metadata-extended": ` specifically the `"lineage": ` parameter?

**Reply by hugoledoux** [reply]
https://github.com/cityjson/metadata-extended/issues
